Framework/Algorithms/src/PolarizationEfficiencyCor.cpp

```diff
@@ -109,13 +112,34 @@ void PolarizationEfficiencyCor::init() {
   declareProperty(Prop::FLIPPERS, "", std::make_shared<Kernel::ListValidator<std::string>>(setups),
                   "Flipper configurations of the input workspaces  (Wildes method only)");
 
+  setPropertySettings(Prop::FLIPPERS, std::make_unique<Kernel::EnabledWhenProperty>(
```

Comment:
I think the changes to this file have been staged by mistake? It would be better to edit this commit (rather than making a new one removing the changes) to keep the history nice. 

`git reset HEAD^` to uncommit the most recent commit and `git add -i` to only add the relevant files may be useful here.